### DISCUSSION TITLE
Set minimum iOS version in build metadata

### DISCUSSION THREAD

**Discussion by siticum** [buildnatively/community#80] [Ideas]
### Use case

```Text
Our app is running a MapPlugin on a very tailored UI/UX with effects. This affects app UI performance and causes crashes on older devices. We would like to define minimum iOS version on the builds from the dashboard to control the distribution and make sure the app is giver its users the best possible experience.

Same goes for Vision and other devices versions.
```

### Idea

```Text
Add a page and field to choose minimum iOS version and include this in the builds. Maybe its possible to get the version list from an API?
```

**Comment by romanfurman6** [comment]
Hello @siticum 

You can get OS version from a device by using this: https://docs.buildnatively.com/guides/integration/device

**Reply by siticum** [reply]
Hi Roman.

The purpose is limiting the availability in the App Store so that older devices can’t download the app. Finding the iOS version won’t do, as it will still be available.

Device limitation in not possible in Apple Developer, but it’s possible to limit it to a minimum iOS version. Older devices can’t update to newer iOS versions which means naturally exclude the old iPhones.

 Being able to set minimum iOS in natively app settings and write this to the Build meta data will be great!

**Reply by romanfurman6** [reply]
Hey

The minimum version right now is iOS 13 (the oldest deice iPhone 6S) 

I think we would consider adding these in the future updates
